includes/models/model.llms.quiz.php

```diff
@@ -213,7 +226,72 @@ public function can_be_resumed_by_student( $user_id = null ) {
 			$can_be_resumed_by_student = $last_attempt && $last_attempt->can_be_resumed();
 		}
 
-		return $can_be_resumed;
+		// Check if the quiz can be resumed by a student and if the resume time limit is not reached.
+		if ( $can_be_resumed_by_student && $this->has_resume_attempt_time_expired() ) {
+			$can_be_resumed_by_student = false;
+		}
+
+		return $can_be_resumed_by_student;
+
+	}
+
+	/**
+	 * Determine if the student's resume quiz attempt time limit is expired.
+	 *
+	 * The resume quiz attempt time limit for a student is 1 day.
+	 *
+	 * @since [version]
+	 *
+	 * @return bool
+	 */
+	private function has_resume_attempt_time_expired() {
```

Comment:
I think this is an attempt method,
And `LLMS_Quiz_Attempt::can_be_resumed() ` can call it.

Reply:
Moved the function in `model.llms.quiz.attempt.php`.